server/app.js

```diff
@@ -0,0 +1,11 @@
+const server = require('./server');
+
+//const port = config.port;
+const port = 8000;
+server.listen((err) => {
+	if (err) {
+		console.log(`API error: ${err}`);
+		return;
+	}
+	console.log(`API is running on port ${port}`);
+});
```

Comment:
This code is pretty clean and obvious Edward...but consider commenting your code